.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 'Bug report'
+labels: Bug
+assignees: bep
+about: Create a report to help us improve
+---
+**Describe the bug**
```

Comment:
Remove all body in this issue template for now. Mostly because I hate these "fill in the blanks" when I see them in other projects. Either you know how to write a good bug report or you don't, and the generic template here does not match the Hugo world particularly well, so we might as well give the user a blank sheet.